src/stores/QueryStore.ts

```diff
@@ -201,13 +201,24 @@ export default class QueryStore extends Store<QueryStoreState> {
                     queryText: queryText,
                 }
             })
+            // make sure there are always at least two input boxes
+            while (queryPoints.length < 2) {
```

Comment:
Previously we handled this in NavBar where it was completely unrelated and did not even work for `point=queryString`.